--- origsrc/dbus-1.2.12/configure.in	2009-02-17 00:32:08.778094600 -0600
+++ src/dbus-1.2.12/configure.in	2009-02-18 03:51:48.621625000 -0600
@@ -284,7 +284,9 @@ if test "x$GCC" = "xyes"; then
   esac
   case " $CFLAGS " in
   *[\ \	]-Wno-pointer-sign[\ \	]*) ;;
-  *) CFLAGS="$CFLAGS -Wno-pointer-sign" ;;
+  *) if cc_supports_flag -Wno-pointer-sign; then
+        CFLAGS="$CFLAGS -Wno-pointer-sign" 
+     fi
   esac  
   
   # http://bugs.freedesktop.org/show_bug.cgi?id=19195
